reboot_required/manifest.json

```diff
@@ -0,0 +1,20 @@
+{
+  "maintainer": "help@datadoghq.com",
```

Comment:
If you accept to be the maintainer, this would need to be updated to one of your email address.

Reply:
Sure no problem. I'll update this PR.